requirements.txt

```diff
@@ -1,6 +1,28 @@
 asgiref==3.2.10
+certifi==2020.6.20
+cffi==1.14.3
+chardet==3.0.4
+cryptography==3.1.1
+defusedxml==0.7.0rc1
 Django==3.1.2
+django-crispy-forms==1.9.2
+django-extensions==3.0.9
+django-filter==2.4.0
+djangorestframework==3.12.1
+idna==2.10
+Markdown==3.2.2
+oauthlib==3.1.0
 Pillow==7.2.0
 psycopg2-binary==2.8.6
+pycparser==2.20
+PyJWT==1.7.1
+python-decouple==3.3
+python3-openid==3.2.0
 pytz==2020.1
-sqlparse==0.3.1
+requests==2.24.0
+requests-oauthlib==1.3.0
+six==1.15.0
+social-auth-app-django==4.0.0
+social-auth-core==3.3.3
+urllib3==1.25.10
+sqlparse==0.3.1
```

Comment:
Remove unnecessary requirements, the server is well running with existing requirements.

Reply:
Done